src/Orleans.Core/Configuration/ProviderConfiguration.cs

```diff
@@ -115,7 +103,7 @@ internal void Load(XmlElement child, IDictionary<string, IProviderConfiguration>
                 }
             }
 
-            LoadProviderConfigurations(child, nsManager, alreadyLoaded, c => childConfigurations.Add(c));
+            LoadProviderConfigurations(child, nsManager, alreadyLoaded, c => { });
```

Comment:
Should we drop the last parameter from the method?

Reply:
wanted to.. it still has legit uses in other parts of the code :/

Reply:
🤮